cmake/partition_manager.cmake

```diff
@@ -185,6 +185,19 @@ if (CONFIG_PM_EXTERNAL_FLASH)
     )
 endif()
 
+if (CONFIG_NRF53_UPGRADE_NETWORK_CORE
+  AND CONFIG_HCI_RPMSG_BUILD_STRATEGY_FROM_SOURCE
+  AND (mcuboot_UPDATEABLE_IMAGE_NUMBER GREATER 1))
+  add_region(
+    NAME ram_flash
+    SIZE 0x40000
+    BASE 0x00000000
```

Comment:
The base address also has a kconfig symbol. Note that both of these are derived from DT

Reply:
The base address used here is fake since `flash_simulator` does not use a real base address since it stores things in a global array called `mock_flash`. I'll add Kconfig with hooks into the `flash_ctrl` device tree node.